autoload/lsp/completion.vim

```diff
@@ -92,20 +92,30 @@ export def CompletionReply(lspserver: dict<any>, cItems: any)
     lspserver.completeItemsIsIncomplete = cItems.isIncomplete
   endif
 
+  # Get the keyword prefix before the current cursor column.
+  var chcol = charcol('.')
+  var starttext = chcol == 1 ? '' : getline('.')[ : chcol - 2]
+  var [prefix, start_idx, end_idx] = starttext->matchstrpos('\k*$')
+  if opt.lspOptions.completionMatcher == 'icase'
+    prefix = prefix->tolower()
+  endif
+
+  var start_col = start_idx + 1
+
   if opt.lspOptions.ultisnipsSupport
       call UltiSnips#SnippetsInCurrentScope(1)
```

Comment:
I would like to eventually move the Ultisnips specific code to a separate file.  We can may be use a snippets subdirectory and add snippet plugin specific files there.

Reply:
Ok, in next pull request I try to relocate it to separate file. But now I want find a solution for this https://github.com/yegappan/lsp/issues/215 and I don`t now how long it will take